resteasy-client-microprofile/pom.xml

```diff
@@ -23,5 +23,26 @@
             <artifactId>resteasy-client</artifactId>
             <version>${project.version}</version>
         </dependency>
+        
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-cdi</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        
+        <dependency>

```

Comment:
resteasy-cdi has dependency on this - https://github.com/resteasy/Resteasy/blob/master/resteasy-cdi/pom.xml#L29

Is it needed to be explicitly declared here ?

Reply:
The MP* classes are needed to have a custom version of the ClientInvocation and ResteasyProviderFactory , which included changes for the MP Rest Client.
I'll check the cdi dependency thing later/tomorrow, thanks

Reply:
OK, checked, that dependency is needed because the javax.enterprise:cdi-api dependency defaults to provided scope due to the way it's declared in resteasy-dependencies-bom/pom.xml